nestjs-counter/apps/worker/src/exchange-rates-worker/exchange-rates-worker.providers.ts

```diff
@@ -0,0 +1,33 @@
+import { ActivitiesService } from '../activities/activities.service';
+import { DefaultLogger, LogEntry, Runtime, Worker } from '@temporalio/worker';
+import { taskQueue } from '@app/shared';
+
+export const exchangeRatesWorkerProviders = [
+  {
+    provide: 'EXCHANGE_RATES_WORKER',
+    inject: [ActivitiesService],
+    useFactory: async (activitiesService: ActivitiesService) => {
+      Runtime.install({
+        logger: new DefaultLogger('WARN', (entry: LogEntry) =>
+          console.log(`[${entry.level}]`, entry.message),
+        ),
+      });
+
+      const activities = {
+        getExchangeRates:
+          activitiesService.getExchangeRates.bind(activitiesService),
+      };
+
+      const worker = await Worker.create({
+        workflowsPath: require.resolve('../temporal/workflows'),
```

Comment:
I'd show how to prove the native connection and more configuration like task queue / namespace.

Reply:
Still think this is relevant.